checkov/terraform/graph_builder/graph_components/module.py

```diff
@@ -200,6 +201,9 @@ def _add_resource(self, blocks: List[Dict[str, Dict[str, Any]]], path: TFDefinit
                     provisioner = attributes.get("provisioner")
                     if provisioner:
                         self._handle_provisioner(provisioner, attributes)
+                    for reserved_attribute in reserved_attributes_to_scan:
```

Comment:
separate to function for readability  + in line 207 use `CustomAttributes.Resource_TYPE`

Reply:
updated 👍